modules/accounting/assets/src/admin/components/invoice/InvoiceCreate.vue

```diff
@@ -48,7 +48,7 @@
             <div class="wperp-table-responsive">
                 <!-- Start Invoice Items Table -->
                 <div class="table-container">
-                    <table class="wperp-table wperp-form-table">
+                    <table class="wperp-create-invoice-table wperp-form-table">
```

Comment:
Why do you need to change the default `wperp-table` class and introduce a new one?